modoboa/core/migrations/0021_merge_20200429_1340.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.10 on 2020-04-29 13:40
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0020_localconfig_need_dovecot_update'),
+        ('core', '0020_auto_20200421_0851'),
+    ]
+
+    operations = [
+    ]
```

Comment:
Instead of a merge migration, please rename your 0020_localconfig_need_dovecot_update to 0021_localconfig_need_dovecot_update and update the dependencies inside to 0020_auto_20200421_0851.